app/src/main/java/ch/epfl/sweng/hostme/ui/messages/ChatActivity.java

```diff
@@ -48,12 +47,15 @@ public class ChatActivity extends AppCompatActivity {
     private ChatAdapter chatAdapter;
     private ImageView launchButt;
     private String conversionId = null;
```

Comment:
I was a little confused by everything in the code being labeled `conversion...` but I quickly realized that it means "conversation". Perhaps we will take a quick moment to change that to `conversation...` or `convo...` for readability in a future PR.